fmriprep/workflows/base.py

```diff
@@ -335,24 +372,23 @@ def init_single_subject_wf(subject_id):
         )
 
         if config.workflow.use_syn_sdc and not fmap_estimators:
-            message = ("Fieldmap-less (SyN) estimation was requested, but "
-                       "PhaseEncodingDirection information appears to be "
-                       "absent.")
+            message = (
+                "Fieldmap-less (SyN) estimation was requested, but "
+                "PhaseEncodingDirection information appears to be "
+                "absent."
+            )
             config.loggers.workflow.error(message)
             if config.workflow.use_syn_sdc == "error":
                 raise ValueError(message)
 
-        if (
-            "fieldmaps" in config.workflow.ignore
-            and [f for f in fmap_estimators
-                 if f.method != fm.EstimatorType.ANAT]
-        ):
+        if "fieldmaps" in config.workflow.ignore and [
+            f for f in fmap_estimators if f.method != fm.EstimatorType.ANAT
+        ]:
```

Comment:
I think this should be `==`. Does this match a known bug? Also, either way, it seems cleaner to use `any()` than to make a list and check it's non-empty.

```suggestion
        if "fieldmaps" in config.workflow.ignore and any(
            f.method == fm.EstimatorType.ANAT for f in fmap_estimators
        ):
```

Reply:
I think you're right, this looks like a bug